src/main/kotlin/lotto/domain/Store.kt

```diff
@@ -0,0 +1,17 @@
+package lotto.domain
+
+object Store {
+    fun purchase(price: Int): MutableList<Lotto> {
+        validatePurchasePrice(price)
+
+        return issueLottos(price / Policy.LOTTO_PRICE)
+    }
+
+    private fun validatePurchasePrice(price: Int) {
+        require(price % Policy.LOTTO_PRICE == 0) { "로또 금액은 1000원 단위로 입력할 수 있습니다." }
+    }
+
+    private fun issueLottos(count: Int): MutableList<Lotto> {
+        return MutableList(count) { LottoGenerator.generate() }
+    }
+}
```

Comment:
가변 타입을 사용하지 않아도 되는 곳에서는 불필요한 side effect 방지를 위해 불변 자료형을 사용하는게 좋습니다!
```suggestion
object Store {
    fun purchase(price: Int): List<Lotto> {
        validatePurchasePrice(price)

        return issueLottos(price / Policy.LOTTO_PRICE)
    }

    private fun validatePurchasePrice(price: Int) {
        require(price % Policy.LOTTO_PRICE == 0) { "로또 금액은 1000원 단위로 입력할 수 있습니다." }
    }

    private fun issueLottos(count: Int): List<Lotto> {
        return List(count) { LottoGenerator.generate() }
    }
}
```